src/test/bridges/uniswap/UniswapBridgeUnit.t.sol

```diff
@@ -0,0 +1,325 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Aztec.
+pragma solidity >=0.8.4;
+
+import {Test} from "forge-std/Test.sol";
+import {AztecTypes} from "../../../aztec/libraries/AztecTypes.sol";
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {ErrorLib} from "../../../bridges/base/ErrorLib.sol";
+import {UniswapBridge} from "../../../bridges/uniswap/UniswapBridge.sol";
+import {IQuoter} from "../../../interfaces/uniswapv3/IQuoter.sol";
+
+contract UniswapBridgeUnitTest is Test {
```

Comment:
Would be useful to add USDT as an asset used in the testing as it doesn't follow the IERC20 interface and at the same time have <18 decimals.

Reply:
Addressed in d7023ac58dfcb9fed141b3d440b0146dea11e49a